RecoPixelVertexing/PixelTriplets/plugins/BuildFile.xml

```diff
@@ -2,8 +2,16 @@
 <use   name="RecoPixelVertexing/PixelTriplets"/>
 <use   name="RecoTracker/TkSeedingLayers"/>
 <use   name="RecoPixelVertexing/PixelTrackFitting"/>
-<library   file="*.cc" name="RecoPixelVertexingPixelTripletsPlugins">
+<library   file="*.cu *.cc" name="RecoPixelVertexingPixelTripletsPlugins">
+  <use   name="cuda"/>
   <flags   EDM_PLUGIN="1"/>
+  <flags   CUDA_FLAGS="--expt-relaxed-constexpr"/>
```

Comment:
@fwyzard commented earlier that this is not needed as the flag is included by default in `CUDA_FLAGS`.